src/app/zap-templates/zcl/data-model/chip/electrical-power-measurement-cluster.xml

```diff
@@ -76,23 +74,21 @@ limitations under the License.
         <feature name="ALTC"/>
       </optionalConform>
     </attribute>
-    <attribute code="0x0007" side="server" name="ApparentCurrent" define="APPARENT_CURRENT" type="amperage_ma" isNullable="true" min="0" max="4611686018427387904" optional="true">
+    <attribute code="0x0007" side="server" name="ApparentCurrent" define="APPARENT_CURRENT" type="amperage_ma" isNullable="true" min="-4611686018427387904" max="4611686018427387904" optional="true">
```

Comment:
what are these min/max? These are totally unreadable. Can alchemy fix that? do we really need this kind of precision on what the range is?

Previous 0 seems more reasonable maybe (can the current be negative?)

Reply:
That one looked weird to me too, however the rest of Attributes (like Voltage, ActiveCurrent, ReactiveCurrent...) also have similar extreme values and this was the only one different.

In the spec they are defined as -2^62 to 2^62. Probably this can be changed to a different representation to make it easier to read (as long as it can be parsed and processed properly, I guess).

![image](https://github.com/user-attachments/assets/a98782f9-dccf-4072-b5d5-158cdb763703)